PrabhaK/03 - JavaScript/project/Florist/order.css

```diff
@@ -0,0 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=Bellefair&family=Instrument+Sans:ital,wght@0,400..700;1,400..700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Rethink+Sans:ital,wght@0,400..800;1,400..800&display=swap');
+
+
+@import url('https://fonts.googleapis.com/css2?family=Lacquer&family=Love+Light&display=swap');
+
+
+*{
+    box-sizing:border-box;
+    margin:0;
+       padding:0;
+   }
+   body{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    background-image: url(images/img12.jpg);
+    background-size:cover;
+    color:rgb(0, 0, 0);
+
+    
+       
+}
+.whole-data{
+    margin:3rem 0 3rem 0;
```

Comment:
Do not hard code the values.
create a theme file (default.theme).
create variables under the theme file.
use the variables instead of hardcoded values.


<h3>APPLY THIS TO ALL PLACES.</h3>